repo/packages/A/artifactory/0/resource.json

```diff
@@ -0,0 +1,50 @@
+{
+    "images": {
+        "icon-small": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_sml.png",
+        "icon-medium": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_med.png",
+        "icon-large": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_lrg.png"
+    },
+    "assets": {
+        "container": {
+            "docker": {
+                "artifactory-pro": "jainishshah17/artifactory:latest",
+                "artifactory-ha-primary": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-ha-primary:4.8.2",
+                "artifactory-ha-secondary": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-ha-secondary:4.8.2"
+            }
+        }
+    },
+    "cli":{
```

Comment:
This is a Universe 3.x feature, so we'll need to bump the `packagingVersion`. But I think this requires a DC/OS-specific subcommand CLI, not a generic CLI :/